src/utils.jl

```diff
@@ -0,0 +1,184 @@
+# todo define methods and remove many of the function below to use multiple dispatch
+
+immutable Double2
+x::Float64
```

Comment:
These lines should be indented. We could alternatively opt to use @simonbyrne's DoubleDouble.jl here.